manifests/overlays/prod/configs/argo_cm/resource.inclusions

```diff
@@ -386,6 +400,40 @@
   - ImageTag
   clusters:
   - https://api.ocp4.prod.psi.redhat.com:6443
+- apiGroups:
+  - logging.openshift.io
+  kinds:
+  - Elasticsearch
+  - Kibana
+  clusters:
+  - https://api.ocp4.prod.psi.redhat.com:6443
+- apiGroups:
+  - mongodb.com
+  kinds:
+  - MongoDB
+  - MongoDBUser
+  - MongoDBOpsManager
+  clusters:
+  - https://api.ocp4.prod.psi.redhat.com:6443
+- apiGroups:
+  - tekton.dev
+  kinds:
+  - Condition
+  - PipelineResource
+  - PipelineRun
+  - Pipeline
+  - TaskRun
+  - Task
+  clusters:
+  - https://api.ocp4.prod.psi.redhat.com:6443
+- apiGroups:
+  - triggers.tekton.dev
+  kinds:
+  - EventListener
+  - TriggerBinding
+  - TriggerTemplate
+  clusters:
+  - https://api.ocp4.prod.psi.redhat.com:6443
```

Comment:
this is duplicate i feel, here: https://github.com/AICoE/aicoe-cd/blob/1a6dfa8f1b7f924f3c7b40f226a0799ab748c129/manifests/overlays/prod/configs/argo_cm/resource.inclusions#L9-L27

also, we exclude PipelineRun and Task runs as it would get accumulated in large numbers in the argocd application dashboard.

Reply:
removed the duplication, and pipelinerun / tassrun